src/server.ts

```diff
@@ -16,6 +19,86 @@ import mime from 'mime-types';
 import fs from 'fs/promises';
 import { z } from 'zod';
 
+const execPromise = util.promisify(exec);
+
+export const serverRunId = randomUUID();
+export const activeSandboxContainers = new Map<string, number>();
+
+const containerTimeoutSeconds = parseInt(
```

Comment:
Shall we have something like a config object which gets validated with zod and returns a save and complete config object? It might expose something like `config.containerTimeoutSeconds`